core/broadcast.go

```diff
@@ -15,9 +15,18 @@ import (
 	"github.com/drand/kyber/share/dkg"
 )
 
-// broadcast implements a very simple broadcasting mechanism: for each new packet
-// seen, rebroadcast it once. While this protocol is simple to implement, it
-// does not guarantees anything about the timing of which nodes is going to
+// Broadcast is an interface that represents the minimum functionality required
+// by drand to both (1) be the interface between drand and the dkg logic and (2)
+// implement the broadcasting mechanisn.
+type Broadcast interface {
+	dkg.Board
```

Comment:
this makes for a very 'wide' interface. is it possible to have a `GetBoard()` method in the interface that returns a `dkg.Board` rather than saying that as `Broadcast` is a `Board`?